superset-frontend/plugins/legacy-plugin-chart-map-box/src/controlPanel.ts

```diff
@@ -224,6 +229,8 @@ const config: ControlPanelConfig = {
               label: t('Map Style'),
               clearable: false,
               renderTrigger: true,
+              freeForm: true,
+              validators: [validateMapboxStylesUrl],
```

Comment:
Do you think we should add a link to mapbox styles docs in the tooltip now that the user can paste anything in here?

Reply:
I added a link to Mapbox's documentation below